deploy/vagrant/conf/tachyon.yml

```diff
@@ -17,3 +17,10 @@ Release:
 #  for virtualbox, machine memory is configured in conf/init.yml:Memory
 #  for aws, machine memory is decided by instance type
 WorkerMemory: 1GB
+
+# Number of Tachyon masters in the cluster, should be at least 1.
+# The cluster will be in fault tolerant mode if the number is larger than 1.
+# When it is in fault tolerant mode, the masters will be named as TachyonMaster, TachyonMaster1, TachyonMaster2, etc,
+# currently, zookeeper is installed in /zookeeper and is a single node deployment, its server process runs on TachyonMaster.
+# This option has no effect currently when Tachyon cluster is run through mesos.
```

Comment:
This doesn't work with mesos (and yarn) because neither of them support starting multiple masters? Maybe this comment shouldn't be about only mesos (since yarn integration is also coming), but more about what it does work with.